programs/mango-v4/src/serum3_cpi.rs

```diff
@@ -155,6 +156,20 @@ impl OpenOrdersSlim {
             referrer_rebates_accrued: oo.referrer_rebates_accrued,
         }
     }
+    pub fn from_oo_v2(oo: &OpenOrdersV2, base_lot_size: u64, quote_lot_size: u64) -> Self {
+        let bids_base_lots: u64 = oo.position.bids_base_lots.try_into().unwrap();
+        let asks_base_lots: u64 = oo.position.asks_base_lots.try_into().unwrap();
+        let base_locked_native = bids_base_lots * base_lot_size;
+        let quote_locked_native = asks_base_lots * quote_lot_size;
+
+        Self {
+            native_coin_free: oo.position.base_free_native,
+            native_coin_total: base_locked_native + oo.position.base_free_native,
+            native_pc_free: oo.position.quote_free_native,
+            native_pc_total: quote_locked_native + oo.position.quote_free_native,
+            referrer_rebates_accrued: oo.position.referrer_rebates_available,
```

Comment:
What do we do about `locked_maker_fees`? It sounds like they are locked when a bid is placed; consumed on fill and returned on cancel?

If so, it sounds like they need to be accounted for (I don't think they are for serum3, but it's more important in openbookv2 since market creators can decide fees, as far as I'm aware). When we compute health for the "all asks execute" and "all bids execute" cases, that's the place where fees either got consumed or reimbursed.